syntax = "proto3";
package timestream;

service Timestream {
  rpc getRecordListResponse(RecordListRequest) returns (RecordListResponse);
}

message Record {
  string r = 1;
  string a = 2;
  string h = 3;
  string t = 4;
  string u = 5;
  double c = 6;
  double m = 7;
}

message RecordListRequest {
  uint32 count = 1;
}

message RecordListResponse {
  repeated Record records = 1;
}